README.md

```diff
@@ -47,31 +48,62 @@ const appReducer = combineReducers({
 const store = createStore(
   appReducer
 );
-
 ```
 
 ## 🎛️ API
 
-### `reducers`
+### Overview
 
-The default export is an object of reducers meant to be included in a call to
-Redux's [`combineReducers`](https://redux.js.org/api/combinereducers) when
-setting up the store. Included reducers are `browser`, `error`, `errorReporting`, `intl`,`redirection` and `rendering`.
+[Ducks](https://github.com/erikras/ducks-modular-redux) include the reducer, selectors and action
+creators for a given branch of state. These ducks make up the core of One App's state.
 
-```js
-import { combineReducers } from 'redux';
-import globalReducers from '@americanexpress/one-app-ducks';
+The default export is an object of reducers meant to be included in a call to Redux immutable's
+[`combineReducers`](https://redux.js.org/api/combinereducers) when setting up the store. Included
+reducers are `browser`, `error`, `errorReporting`, `intl`, `redirection` and `rendering`.
 
-const appReducer = combineReducers({
-  ...globalReducers,
-  ...otherReducers, // Your specific reducers
-});
+**Contents:**
+
+- [`browser`](#browser-duck)
+- [`error`](#error-duck)
+- [`errorReporting`](#errorreporting-duck)
+- [`intl`](#intl-duck)
+- [`redirection`](#redirection-duck)
+- [`rendering`](#rendering-duck)
+
+### `browser` Duck
+
+This duck is for reading information about the user's browser. It is particularly helpful on the
+server.
```

Comment:
Can you expand on that second sentence? Maybe `... on the server where this information is not readily available.`?